Documentation/Signoffs/ResistanceSystem.md

```diff
@@ -0,0 +1,73 @@
+Team 4 – Mario Kart & Ride Replay Simulator
+Team Members: Ray Durlin, Blake Pickett, Tyler Chittum, Benjamin Reed, and Sage Mooneyham
+Detail Design: Resistance System
+
+
+Function of the Subsystem: 
+The function of the resistance system is to wirelessly receive an input value from the RRS processing subsystem. The input value is to be received by the H-Bridge controller. The H-Bridge controller will send a power signal to the linear actuator in the form of a pulse width modulated value to adjust the position of the magnets across the flywheel resulting in the creation of eddy currents in the flywheel. These eddy currents create a magnetic field that opposes the source magnetic field. This causes power dissipation in the flywheel, as well as opposing torque, which creates a change of resistance felt by the user. 
+
+Constraints:
+Cost: The main constraint for the resistance system is the cost of materials for both the flywheel, linear motor, and magnets. The total budgeted cost of the resistance system is 50% of the project budget, equaling $600. 
+-	Flywheel: A traditional spin bike flywheel used commercially ranges from $270 to $475 used. To mitigate the cost of the flywheel, team 4 will construct a flywheel using the existing bike tire and aluminum sheet material.
+-	Linear Motor: An ideal linear motor would have specifications of 100% duty cycle and 23.6 inches per sec speed but would cost over $1,000. To mitigate the cost, the team will suffice with a lower duty cycle linear motor while maintaining a speed of 3.3 inches per sec. The cost is approx. $184. The controller will need to manage the power supplied to the linear actuator to ensure the duty cycle rating is not exceeded.
+-	Magnets: A N52 neodymium magnet with a surface field rating of 4000 Gauss or greater ranges from $35 to $50 depending on thickness. Based on the calculations made in the analytical section, the number of magnets required for a rectangular magnet with the dimensions of 2 inches in length by 1 inch in width, with a thickness of 0.5 inch that is 8. The magnet dimensions that are chosen in the resistance system based on cost.
+
+Material:
+- Flywheel Material: The material for the constructed flywheel is a constraint, as a ferromagnetic flywheel will experience magnetic pull when in the magnetic field created by the magnets. Because of this, Team 4’s constructed flywheel will be made of aluminum sheets, a non-ferromagnetic material.
+
+Buildable schematic:
+
+![image](https://user-images.githubusercontent.com/114370750/201011044-d22a8e48-4f29-43f6-9733-91bda5719795.png)
+
+![image](https://user-images.githubusercontent.com/114370750/201011097-6f954d7b-ff05-4396-bd05-49222ed99d69.png)
+
+
+
+
+Analysis:
```

Comment:
Until the constraints are updated I can't provide feedback on the analysis. The analysis should be an analysis of the design showing that it should be able to meet the constraints. But since the constraints don't address significant core functionality, this must be addressed before feedback is given on analysis.